apps/api/src/services/db/userDataService.ts

```diff
@@ -142,6 +150,24 @@ export async function getSettingsOrInit({ anonymousUserId, userId, wantedUsernam
   ]);
 }
 
+async function tryToTransferWallet(prevUserId: string, nextUserId: string) {
+  if (process.env.BILLING_ENABLED !== "true") {
+    return;
+  }
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const { UserWalletRepository } = require("@src/billing/repositories/user-wallet/user-wallet.repository");
```

Comment:
I guess we can add a TODO to refactor this file and make it like the services you built with DI

Reply:
https://github.com/akash-network/console/issues/322